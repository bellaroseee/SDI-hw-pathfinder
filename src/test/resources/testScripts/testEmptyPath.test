# Finding a path to itself

CreateGraph graph1

AddNode graph1 A
AddNode graph1 B
AddNode graph1 C

FindPath graph1 A A

FindPath graph1 B B

FindPath graph1 C C


CreateGraph graph2

AddNode graph2 A
AddNode graph2 B
AddNode graph2 C

FindPath graph2 A A

FindPath graph2 B B

FindPath graph2 C C