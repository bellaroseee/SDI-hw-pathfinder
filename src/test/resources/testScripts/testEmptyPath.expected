# Finding a path to itself

created graph graph1

added node A to graph1
added node B to graph1
added node C to graph1

path from A to A:
total cost: 0.000

path from B to B:
total cost: 0.000

path from C to C:
total cost: 0.000


created graph graph2

added node A to graph2
added node B to graph2
added node C to graph2

path from A to A:
total cost: 0.000

path from B to B:
total cost: 0.000

path from C to C:
total cost: 0.000